io500.sh

```diff
@@ -59,8 +59,8 @@ function setup_paths {
   io500_ior_cmd=$PWD/bin/ior
   io500_mdtest_cmd=$PWD/bin/mdtest
   io500_mdreal_cmd=$PWD/bin/md-real-io
-  io500_mpirun="mpirun"
-  io500_mpiargs="-np 2"
+  io500_mpirun="jsrun"
+  io500_mpiargs="-n 6 -r 3 -a 1 -c 1"
```

Comment:
Are these lines correct?  I don't think these are appropriate for defaults.  Can you change these back please?